presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcStorageManager.java

```diff
@@ -333,6 +325,9 @@ private ShardInfo createShardInfo(UUID shardUuid, File file, Set<String> nodes,
             return shardDelta(shardUuid, Optional.empty());
         }
 
+        // submit for backup and wait until it finishes
+        backupManager.submit(newShardUuid, input).join();
```

Comment:
`join()` wraps the exception in a `CompletionException`. Using `getFutureValue()` from `MoreFutures` is better since it unwraps the exception.